ci/ray_ci/serve.tests.yml

```diff
@@ -0,0 +1 @@
+flaky_tests: []
```

Comment:
For my own education, what will happen to tests marked flaky going forward?

Reply:
Technically, they will just become non-blocking. This is a way for team/test owner to unblock themselves if their tests suddenly become failing and blocking their PR without their faults. I want to see if test owners can manage their tests effectively this way, as well as not abusing this feature too much. If not, or if they want to make thing easier for them to manage, we can enforce through policy and automation down the line.

Reply:
Sounds good! I'll be happy to keep this list empty for the serve tests :D

Reply:
fantastic!